include/staff/slaplans.inc.php

```diff
@@ -36,7 +36,7 @@
 
 $pageNav = new Pagenate($count, $page, PAGE_LIMIT);
 $pageNav->setURL('slas.php', $qs);
-$showing = $pageNav->showing().' '._N('SLA plan', 'SLA plans', $count);
+$showing = $pageNav->showing().' '._N('Service Level Agreement', 'Service Level Agreements', $count);
```

Comment:
Personally, I think Service Level Agreement is more correct; however, honestly, after about six hours, I just got worm out of rephrasing text